portal/views/portal.py

```diff
@@ -542,9 +542,10 @@ def legal():
 def terms_and_conditions():
     """ terms-and-conditions of use page"""
     gil = current_app.config.get('GIL')
-    response_text = VersionedResource.fetch_elements(app_text(Terms_ATMA.name_key()))[0]
+    content, _ = VersionedResource.fetch_elements(
+            app_text(Terms_ATMA.name_key()))
```

Comment:
Paul, I added the item in this PR: https://github.com/uwcirg/ePROMs-site-config/pull/16   Let me know if you see any issue.